### DISCUSSION TITLE
About the possibility of using harmony to integrate data

### DISCUSSION THREAD

**Discussion by wu-yc** [stuart-lab/signac#817] [Q&A] [answered]
signac V0.2 supports the harmony (https://satijalab.org/signac/0.2/articles/integration.html), but current version seems not to support harmony. 

Would you kindly inform me how to use harmony to integrate data?

**Comment by timoast** [comment, marked as answer]
Please see the documentation for Harmony: https://portals.broadinstitute.org/harmony/reference/RunHarmony.html

You can still run Harmony on a Seurat object, we just don't have a vignette for this on the current Signac website
